src/components/profile/UserEmailForm/index.jsx

```diff
@@ -1,5 +1,64 @@
 import React from "react";
+import { Col } from "react-bootstrap";
+import EmailControl from "./EmailControl";
+import EmailAdd from "./EmailAdd";
+import "./index.scss";
 
-export default function UserEmailForm() {
-  return <div>UserEmail</div>;
+// this object should come from the component parent
+const USER_EMAILS = [
```

Comment:
will you need to fetch it from firebase? i didn't understand

Reply:
yes